### DISCUSSION TITLE
Welcome to firefox-vertical-tabs Discussions!

### DISCUSSION THREAD

**Discussion by lucaspar** [lucaspar/firefox-vertical-tabs#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vitaly-zdanevich** [comment]
I just want to say that I LOVE your work, this is a dream.

THANK YOU!
